docs/swarm/swarm-tutorial/create-swarm.md

```diff
@@ -0,0 +1,76 @@
+<!--[metadata]>
++++
+title = "Create a Swarm"
+description = "Initialize the Swarm"
+keywords = ["tutorial, cluster management, swarm"]
+[menu.main]
+identifier="initialize-swarm"
+parent="swarm-tutorial"
+weight=12
++++
+<![end-metadata]-->
+
+## Create a Swarm
+
+After you install Docker on your networked machines and start the Docker Engine
```

Comment:
This seems a bit weird to have prereqs here in prose.  Let's make sure we explicitly tell them to start Docker Engine on the nodes in the previous file, and then we can just recap the prereqs here as a short bullet list.